.github/workflows/testing-workflow.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         set -o pipefail
         pip install pip==22.2.2
         pip install flit wheel
-        flit install -s --deps develop
+        flit install -s --deps all  
```

Comment:
why would it need all dependencies for the testing worklow ?

Reply:
needed for the draw part, which uses plotly & igraph